.github/workflows/update_client.yaml

```diff
@@ -0,0 +1,95 @@
+name: Update SDK
+
+# Define the events that trigger the workflow
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - "api/**"
+  pull_request:
+    types: [closed]
+    paths:
+      - "api/**"
+
+jobs:
+  run-commands:
+    # Define the OS and environment the job will run on
+    runs-on: ubuntu-latest
+
+    # Define the steps in the job
+    steps:
+      # Step 1: Checkout the first repository
+      - name: Checkout repository
+        uses: actions/checkout@v3
+        with:
+          repository: wordlift/docs
+          path: "./docs"
+
+      # Step 2: Checkout the second repository
+      - name: Checkout second repository
+        uses: actions/checkout@v3
```

Comment:
Can we update versions here, if you open https://github.com/wordlift/docs/actions/runs/9252847204, you'll see warnings:

<img width="561" alt="image" src="https://github.com/wordlift/docs/assets/11438/bd4f5b12-7071-4728-842d-c12b112d25ff">